.github/workflows/on-release.yaml

```diff
@@ -119,7 +119,8 @@ jobs:
       - container-whalesay
       - container-whalesay2
       - chart-helloworld
-    if: fromJSON(needs.changes.outputs.changes).workflows == 'true'
+    if:  ${{ always() && (contains(fromJSON(["skipped", "success"]), needs.*.result) && fromJSON(needs.changes.outputs.changes).workflows == 'true') }}
```

Comment:
Do we want to test for not containing `failure` or `cancelled` instead?

Reply:
If something upstream ran and failed, or was cancelled by the scheduler we don't want to continue to the repo release.

Reply:
Failure or cancelled will cause downstream jobs to not run automatically as far as I am aware

Reply:
If we don't leave ```skipped``` in it will also just skip this job as far as I know

Reply:
We'll always run because of `always() &&` so we just care about limiting when we don't want to run.

Reply:
The downstream job is only plumbed into the repo version job, so as long as repo version ran successfully then repo release will too.

Reply:
> We'll always run because of `always() &&` so we just care about limiting when we don't want to run.

With this is the solution just removing ```always() &&``` as then ```&& fromJSON(needs.changes.outputs.changes).workflows == 'true'``` kind of takes over from it